framework/encode/openxr_capture_manager.cpp

```diff
@@ -194,6 +194,82 @@ void OpenXrCaptureManager::EndFramePreDispatch(XrSession session, const XrFrameE
     WriteViewRelativeLocationMetadata(session, *frameEndInfo);
 }
 
+OpenXrCaptureManager::OpenXrCaptureManager() : ApiCaptureManager(format::ApiFamilyId::ApiFamily_OpenXR)
+{
+    reentry_state_.reentry_mode = ReentryMode::kThreadDisable; // Note make this configurable in future?
```

Comment:
I would make a TODO to add this as an option.  I want it configurable just to verify if a bug goes away.  Otherwise, we should only have this one option.